native-image-logging-examples/README.md

```diff
@@ -0,0 +1,75 @@
+# Configuring Logging in Native Image
+
+This demo demonstrates how Native Image supports logging, which is using the `java.util.logging.*` API by default. The logger can be initialized at build time with a custom _logging.properties_ configuration file or at runtime. The below examples show both cases.
+
+### Prerequisites
+* [GraalVM](http://graalvm.org)
+* [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/)
+
+## Preparation
+
+1. Download and install the latest GraalVM JDK with Native Image using [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader):
+  ```bash
+  bash <(curl -sL https://get.graalvm.org/jdk)
+  ```
+
+2. Download or clone the repository and navigate into the `native-image-logging-examples` directory:
+  ```bash
+  git clone https://github.com/graalvm/graalvm-demos
+  cd graalvm-demos/native-image-logging-examples
+  ```
+
+There are two Java classes: one for the build-time logger initialization and the second for runtime logger initialization. The logger will be initialized with a custom _logging.properties_ configuration file, which is placed in the same directory as _BuildTimeLoggerInit.java_ and _RuntimeLoggerInit.java_.
+
+## Build-Time Logger Initialization
+
+In this example, the logger will be initialized at build time with a custom _logging.properties_ configuration file, placed in the same repository as _BuildTimeLoggerInit.java_.
+
+1. Compile _BuildTimeLoggerInit.java_ using `javac`:
+
+    ```bash
+    javac BuildTimeLoggerInit.java
```

Comment:
This also implies that the user has added the GraalVM installation to their path. Maybe be explicit in this and following commands?
```suggestion
    $JAVA_HOME/bin/javac BuildTimeLoggerInit.java
```

Reply:
Okay , I will change  it in all following commands!